dist/IO/lib/IO/Socket.pm

```diff
@@ -396,218 +396,468 @@ IO::Socket - Object interface to socket communications
 
 =head1 SYNOPSIS
 
-    use IO::Socket;
+    #!/usr/bin/env perl
```

Comment:
I think it's best to leave shebangs out of synopses (though this is the least problematic one for users to copy-paste, it's problematic to use in CPAN distributions).

Reply:
I will remove that shortly